docs/SpellExample.java

```diff
@@ -293,13 +266,11 @@ public static void main(String[] args) throws Exception {
          * Just to keep things neat and tidy, delete the database and close the client connection
          */
         deleteDB(adminClient, DB_NAME);
+        adminClient.close();
```

Comment:
You must also close the `client` instance. The underlying resources will only be closed once all clients used have been closed.

Reply:
I think the best approach would be to use `try/finally` do to it, or if you're using java 7 `try-with-resources`

Reply:
I refactored the sample to use a try-with-resources.  Let me know if that looks ok.